test/tests.ts

```diff
@@ -63,11 +69,21 @@ export async function setupTest (preset) {
   return ctx
 }
 
-export async function startServer (ctx, handle) {
+export async function startServer (ctx: Context, handle) {
   ctx.server = await listen(handle)
   console.log('>', ctx.server!.url)
 }
 
+export async function startDevServer (ctx: Context) {
+  const devServer = createDevServer(ctx.nitro)
+  ctx.server = await devServer.listen({})
+  await prepare(ctx.nitro)
+  await build(ctx.nitro)
+  // TODO: How to await that dev server is ready?!
+  await new Promise(resolve => setTimeout(resolve, 1000))
```

Comment:
wdyt @pi0?

```suggestion
  const ready = new Promise<void>((resolve) => { ctx.nitro.hooks.hook('dev:reload', () => resolve()) })
  await build(ctx.nitro)
  await ready
```

Reply:
`dev:reload` hook seems a nice idea! Addtionally we need to move this test to dev preset.

Reply:
@manniL would you move this to the dev preset? I think @pi0 just means extracting this function and inlining it in the dev preset test file.

Reply:
Ohh, yes sure 👍🏻